index.html

```diff
@@ -1,22 +1,22 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <title>Mitchell Behrens' CSD120 Website</title>
     <meta charset="UTF-8">
-    <link rel ="stylesheet" href = styles/main.css>
+    <link rel="stylesheet" href="styles/main.css">
```

Comment:
Style recommendation: it's almost universal practice to use NO spaces around the `=` in attributes, and to use quotes around attribute values